test/unit/puppet_class_importer_test.rb

```diff
@@ -55,6 +55,16 @@ def setup
     assert !importer.actual_environments.include?("foreman-testing")
   end
 
+  test "should save parameter with different default_value when importing" do
```

Comment:
This only checks an update in a single environment. Please add a test to check the use-case we discussed (different defaults in different environments).